docker-compose.test.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  test:
+    extends:
+      file: ./docker-compose.yml
+      service: node
+    environment:
+      - AWS_DYNAMODB_ENDPOINT=http://dynamodbtest:8000
+      - NODE_ENV=test
+    links:
+      - dynamodbtest
+    depends_on:
+      - dynamodbtest
+    ports:
+      - 3001:3001
+    command: npm test
```

Comment:
I'd leave the command as is, and instead use docker-compose run test nom test as your testing command on the likes of travis